src/lib/es.js

```diff
@@ -150,13 +149,12 @@ function buildQuery(parameters) {
 }
 
 function buildIdQuery(id) {
-  // TODO: Figure out why term: {id} does not work but {_id} does
   return {
     query: {
-      constant_score: {
+      bool: {
         filter: {
           term: {
-            _id: id
+            id: id
```

Comment:
my guess is that this wasn't working at some point because strings were not being defaulted to `keyword` type, so any punctuation etc in them was causing them to be tokenized and not match search.